joomla/joomla-cms
# [5.0] Update deleted files list in script.php for 5.0.0-beta3

Pull Request for Issue # .

### Summary of Changes

Add deleted files from recent changes to the list of files to be deleted on update in script.php.
- Add missing gzipped media files forgotten with PR #41870 
- Remove wrong file (typo) "/media/system/css/mootree_rtl,min.css" (with comma instead of dot) which was added with the same PR
- Add missing section comment "// From 5.0.0-beta2 to 5.0.0-beta3" to deleted files.
- Add deleted files and folders of indirect dependencies "fgrosse/phpasn1" and "stella-maris/clock" from PR #41862

I don't see any other PRs in the pipeline in 4.4-dev and 5.0-dev which could cause deleted files and folders, so I think this is ready to go.

### Testing Instructions

Code review.

Or update from a current 4.4-dev branch or latest 4.4.0-beta3-dev nightly or a 4.4.0-beta2 to the latest 5.0.0-beta3-dev nightly for the actual result and to the patched update package or custom update URL created by Drone for this PR for the expected result.

### Actual result BEFORE applying this Pull Request

Gzipped media files mentioned in the description above have not been deleted with the update.

File "administrator/components/com_admin/script.php" contains a typo "/media/system/css/mootree_rtl,min.css" (with comma), so the "/media/system/css/mootree_rtl.min.css" (with dot) has also not been deleted.

### Expected result AFTER applying this Pull Request

Gzipped media files mentioned in the description above have been deleted with the update.

File "/media/system/css/mootree_rtl.min.css" has also been deleted.

File "administrator/components/com_admin/script.php" doesn't contain typo "/media/system/css/mootree_rtl,min.css".

### Link to documentations
Please select:
